tsconfig-strict.json

```diff
@@ -31,6 +31,8 @@
     }
   },
   "files": [
+    "core/templates/services/autoplayed-videos.service.ts",
+    "core/templates/services/autoplayed-videos.service.spec.ts",
```

Comment:
Please add these to the other files so that the sorted order is kept.

Reply:
Done :)